app/src/main/res/values/colors.xml

```diff
@@ -9,9 +9,9 @@
     <color name="black">#FF000000</color>
     <color name="white">#FFFFFFFF</color>
     <color name="secundario">#7b1fa2</color>
-    <color name="verde">#388e3c</color>
```

Comment:
This was fine, in colors it should go only colors name like darkGreen, boneWhite or something like that. Never use color names with semantic value